odkx-src/suitcase-intro.rst

```diff
@@ -107,14 +123,40 @@ Uploading to the server
   
   Then modify the :guilabel:`Upload` field to that file path by clicking on the :guilabel:`...` button, and then press :guilabel:`Upload`.
 
+  Screenshot of upload tab:
+
+  .. image:: /img/suitcase-intro/upload_tab.png
+   :width: 600
+
   .. note::
 
       Suitcase GUI supports only Uploading of files and not Updating or Modifying, check out :ref:`Suitcase CLI  <suitcase-cli-update>`
 
   .. _suitcase-gui-reset:
   
 Resetting the server
-  The :guilabel:`Reset` button can be found under the :guilabel:`Upload` tab. Clicking it will reset the the server without any warning or confirmation.
+  The :guilabel:`Reset` button can be found under the :guilabel:`Clear` tab. Clicking it will reset the the server after a warning and a confirmation.
```

Comment:
"Clicking it" is a bit unclear folks might worry it is the clear tab. Suggest "Clicking 'reset'

Reply:
Also should reference clear png here. Make sure all pngs are embedded.